tools/cargo-each/src/check.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::{bail, Result};
+use cargo_metadata::Package;
+
+use crate::config::Config;
+
+pub(crate) fn configuration(package: &Package, config: &Config<'_>) -> Result<()> {
+    sets(package, config)?;
+
+    combinations(package, config)
+}
+
+fn sets(package: &Package, config: &Config<'_>) -> Result<()> {
+    config.sets.iter().try_for_each(|(name, set)| {
+        if set.members.is_empty() {
+            cold();
```

Comment:
premature optimizations may only pollute the code and make reading it more difficult